teams/vsa/engineering/forms-system/readme.md

```diff
@@ -1,17 +1,50 @@
-## VSA internal working guidance on the use of VA Forms System (VAFS) and the VA fork of react-json-schema-form (RJSF)
-* [2020-03-forms-doc-feedback.md](./2020-03-forms-doc-feedback.md) \
-VSA front-end developers had an open discussion about issues they ran into while working with Forms. Here's the candid feedback. 
-  
-* [possible-issues.md](./possible-issues.md) \
-Our current (evolving) list of *possible* issues, things we may need to vett internally.
+# VAFS/RJSF Recipes & Best Practices
+
+The VA Forms System (VAFS) is built upon the VA fork of react-json-schema-form (RJSF). Here is VSA's set of internal guidelines and recipies for common yet more advanced forms you may need to create using VASF/RJSF. The recipies are organized by the form 'task' you are trying to achieve. 
+
+## Style and Organization
+
+- [Recommended Code Structure](./code_structure.md) - A way to organize your code so that it is easy to maintain.
+
+- [Using a Constants File](./constants.md) - Using a central file for all the variables you will use over and over again.
+
+
+## Implementation Patterns
+These real-world implementation patterns are somewhat gnarlky in detail. In some situations we may be using the forms system beyond what it was originally designed for.
+
+- [List Loop](./list_loop.md) - This is needed to build a form that allows a user to input a few items and a separate page for each of those items to enter more information about that respective item.

```

Comment:
@cohnjesse could you cite where in the VSP doc they recommend this? My understanding is that they recommended this pattern but did not provide an example, so yours would be the example.

Reply:
The page is actually missing from the VSP documentation, on [this](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/available-widgets#arraycountwidget) page there is a widget called an ArrayCountWidget, in there you can see a link to `sequential duplicate form groups` and that link is supposed to go to a page that explains this.

Reply:
thanks I submitted a broken link issue in github